CMakeLists.txt

```diff
@@ -180,5 +180,11 @@ add_subdirectory(src)
 add_subdirectory(example/calculate_pi)
 add_subdirectory(example/face_fields)
 add_subdirectory(example/advection)
-
```

Comment:
I would prefer to keep this blank like, to separate the add_subdirectory's from the include.